File: SoloLearn/Hovercraft/Hovercraft.java
import java.util.Scanner;

public class Hovercraft {
    
    public static void main(String[] args) {
        Scanner input = new Scannner(System.in);
        int sales = input.nextInt();
        if (sales == 7)
            System.out.println("Broke Even");
        else if (sales < 7)
            System.out.println("Loss");
        else    
            System.out.println("Profit");
    }
}

File: SoloLearn/Hovercraft/Hovercraft.cpp
#include <iostream>
using namespace std;

int main() {
    int sales = 0;
    cin >> sales;
    if (sales == 7)
        cout << "Broke Even";
    else if (sales < 7)
        cout << "Loss";
    else    
        cout << "Profit";
    return 0;
}